rstudio/c9s-python-3.9/Dockerfile

```diff
@@ -63,7 +63,7 @@ RUN yum -y module enable nginx:$NGINX_VERSION && \
     rpm -V $INSTALL_PKGS && \
     nginx -v 2>&1 | grep -qe "nginx/$NGINX_VERSION\." && echo "Found VERSION $NGINX_VERSION" && \
     # spawn-fcgi is not in epel9
-    rpm -i --nodocs https://www.rpmfind.net/linux/fedora/linux/releases/34/Everything/x86_64/os/Packages/s/spawn-fcgi-1.6.3-20.fc34.x86_64.rpm && \
```

Comment:
This line brokes the image build. The fix has been introduced on this PR using: https://www.rpmfind.net/linux/fedora/linux/releases/36/Everything/x86_64/os/Packages/s/spawn-fcgi-1.6.3-22.fc36.x86_64.rpm